Fix documentation CI failure

In contrast to CDash reporting, CI logs show that failure occurs at the Doxygen build step:
```
2022-03-08T22:09:10.9217174Z [1585/1593] Generating sphinx documentation: html
2022-03-08T22:09:10.9217505Z FAILED: CMakeFiles/build_html 
2022-03-08T22:09:10.9218286Z cd /home/runner/work/bld/ITKEx-build && /home/runner/work/bld/itkpython/bin/sphinx-build -b html -c /home/runner/work/bld/ITKEx-build/Formatting /home/runner/work/bld/ITKEx-build /home/runner/work/bld/ITKEx-build/html
2022-03-08T22:09:10.9218407Z Running Sphinx v4.4.0
2022-03-08T22:09:10.9218418Z 
2022-03-08T22:09:10.9218571Z Extension error (doxylink.doxylink):
2022-03-08T22:09:10.9219106Z Handler <function setup_doxylink_roles at 0x7fe8b838fdc0> for event 'builder-inited' threw an exception (exception: no element found: line 1, column 0)
2022-03-08T22:09:10.9222461Z [1586/1593] Building CXX object src/Segmentation/ConnectedComponents/LabelConnectComponentsInGrayscaleImage/CMakeFiles/LabelConnectComponentsInGrayscaleImage.dir/Code.cxx.o
2022-03-08T22:09:10.9222939Z [1587/1593] Building CXX object src/Segmentation/ConnectedComponents/ExtraLargestConnectComponentFromBinaryImage/CMakeFiles/ExtraLargestConnectComponentFromBinaryImage.dir/Code.cxx.o
2022-03-08T22:09:10.9223085Z ninja: build stopped: subcommand failed.
2022-03-08T22:09:10.9223222Z ninja: build stopped: subcommand failed.
2022-03-08T22:09:10.9223342Z Command exited with the value: 1
```

I am able to recreate this failure locally, will continue to investigate.

--- a/CMake/DownloadDoxygenTAG.cmake
+++ b/CMake/DownloadDoxygenTAG.cmake
@@ -1,4 +1,4 @@
-file( DOWNLOAD https://public.kitware.com/pub/itk/NightlyDoxygen/InsightDoxygenDocTag.gz
+file( DOWNLOAD https://itk.org/files/NightlyDoxygen/InsightDoxygenDocTag.gz
   ${ITKDoxygenTAG_TEMP_DIR}/InsightDoxygen.tag.gz SHOW_PROGRESS
   )
 

--- a/src/Filtering/ImageGrid/CropImageBySpecifyingRegion2/Code.cxx
+++ b/src/Filtering/ImageGrid/CropImageBySpecifyingRegion2/Code.cxx
@@ -107,6 +107,11 @@ CreateImage(ImageType::Pointer image)
   image->Allocate();
   image->FillBuffer(itk::NumericTraits<ImageType::PixelType>::ZeroValue());
 
+  PixelType rgbPixel;
+  rgbPixel[0] = 200;
+  rgbPixel[1] = 200;
+  rgbPixel[2] = 200;
+
   // Make a rectangle, centered at (100,150) with sides 160 & 240
   // This provides a 20 x 30 border around the square for the crop filter to remove
   for (unsigned int r = 20; r < 180; r++)
@@ -117,7 +122,7 @@ CreateImage(ImageType::Pointer image)
       pixelIndex[0] = r;
       pixelIndex[1] = c;
 
-      image->SetPixel(pixelIndex, 200);
+      image->SetPixel(pixelIndex, rgbPixel);
     }
   }
 }

